galley/pkg/config/mesh/const.go

```diff
@@ -12,11 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package meshcfg
+package mesh
 
 import (
 	"istio.io/istio/pkg/config/resource"
 )
 
-// ResourceName for the Istio Mesh Config resource
-var ResourceName = resource.NewFullName("istio-system", "meshconfig")
+// MeshConfigResourceName is the resource name for the Istio MeshConfig resource.
+var MeshConfigResourceName = resource.NewFullName("istio-system", "meshconfig")
```

Comment:
Tell me about this constant "istio-system".

Reply:
You mean you would like to see it as a defined constant?

Reply:
From the CLI we have a `--istioNamespace` parameter.  I am not sure if it is passed to the actual analyzer.  If possible pick it up from the CLI.  If not pick it up as a constant so we can connect it to `--istioNamespace` in the future.

Reply:
This pkg is not only used by analyzer but also galley pipeline (though we are not using it for now.) Yes, it can be an improvement when we do more code consolidation.